### PR TITLE
MYTHOS-819 Clean Up

### PR DESCRIPTION
**Pull request by erik-mythical**
Remove UserState

Remove UpdateItemTypeMetadataRequest metadata

### CODE REVIEW

**Review by kwang-mythical — APPROVED** [review]

**Review by DustinLiang — APPROVED** [review]

**Review by axcrest — APPROVED** [review]

File: streams/user/definition.proto [hunk 1]
```diff
@@ -12,8 +11,6 @@ package saga.rpc.streams.user;
 message UserStatusUpdate {
```
Comment (axcrest) [inline streams/user/definition.proto:11]: Do we need this message at all?

Comment (erik-mythical) [reply streams/user/definition.proto:11]: We probably need something to come back when a user is created.

Comment (axcrest) [reply streams/user/definition.proto:11]: Oh is this the response?  I thought it was to support an updateUser() feature.

Comment (erik-mythical) [reply streams/user/definition.proto:11]: Each client has an update event that is fired when something changes so this covers any write operation, but create is the only one for user.

Comment (axcrest) [reply streams/user/definition.proto:11]: ok cool

**Pull request merged**